.config/nextest.toml

```diff
@@ -1,10 +1,19 @@
 [profile.default]
 slow-timeout = "2m"
-default-filter = 'not test(slow_)'
+default-filter = 'not test(slow_) and not package(tests)'
 retries = 2
 
 [profile.slow]
+slow-timeout = "2m"
```

Comment:
Slow tests take more time than 2 minutes, so I think this timeout should be increased

Reply:
I'll double-check, but I believe this is just the time before a test is marked slow, not the time before a test is killed.